src/js/background/index.js

```diff
@@ -36,6 +37,7 @@ browser.runtime.onInstalled.addListener(details => {
 });
 
 browser.runtime.onMessage.addListener((request, sender, sendResponse) => {
+  console.log('request', request.action);
```

Comment:
wanna remove console.log?

Reply:
Is okay because it's in bg script, wont log to client side